### DISCUSSION TITLE
DYS features

### DISCUSSION THREAD

**Discussion by llemeurfr** [edrlab/thorium-reader#1775] [Ideas]
Interesting features for dyslexic people: 

- Disabling hyphenation.
- Displaying alternate colored lines (with configurable colors).
- Limiting the number of lines visible on the screen (viewport reduction).
- Managing the font and background color. 
- Saving a theme (set of reading options).
- Managing a reading rule moved by the user.
- Word by word TTS on user selection.
- Syllable highlighting.
- Highlighting of punctuation.
- Highlighting of reading units (semantic addition in the text required).   

and, associated with another sub-project (synchronized media):

- Display of images explaining certain words on user action.

**Comment by llemeurfr** [comment]
I keep it there for future reference.

**Reply by christophekede** [reply]
Hello @llemeurfr 
Great job !
I am interested, to make thorium more accessible, but I admit that I have trouble understanding the parts of the code to modify to implement color alternation for example, can you give me a little more informations please ?

**Comment by sjabsr** [comment]
If that might be a ressource, here's the dev guide and repository of a tool that implements syllable colorization :
https://readapt.ai/developer-guide
https://github.com/Contentsquare/readapt
